tests/conftest.py

```
@@ -2510,42 +2510,38 @@ def dvs_for_upgrade(
     updated_default_storage_class_ocs_virt,
+    golden_images_namespace,
 ):
-    golden_images_namespace_name = py_config["golden_images_namespace"]
     dvs_list = []
-    artifactory_secret = utilities.artifactory.get_artifactory_secret(namespace=golden_images_namespace_name)
-    artifactory_config_map = utilities.artifactory.get_artifactory_config_map(namespace=golden_images_namespace_name)
+    with artifactory_credentials(
+        namespace=golden_images_namespace.name, client=golden_images_namespace.client
+    ) as artifactory:
+        for sc in py_config["storage_class_matrix"]:
+            storage_class = [*sc][0]
```

Ruff (0.16.0)

[warning] Prefer `next(iter(sc))` over single element slice

Replace with `next(iter(sc))`

(RUF015)